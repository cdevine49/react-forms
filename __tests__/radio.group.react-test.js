import React from 'react';
import RadioGroup from '../src/components/radio/group';
import renderer from 'react-test-renderer';

describe('RadioGroup', () => {
  test('Default', () => {
    const tree = renderer.create(
      <RadioGroup
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});