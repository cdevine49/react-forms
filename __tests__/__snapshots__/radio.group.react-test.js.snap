// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioGroup Default 1`] = `
<fieldset
  className="radio-button-container"
/>
`;
